src/Node/GethNode.ts

```diff
@@ -59,20 +59,35 @@ export default class GethNode extends Node {
   public readBootnodes(): void {
     // Added try catch, because this is called even in case of mosaic stop.
     // This is needed only while mosaic start.
+
+    let bootNodePath = this.originChain
+      ? path.join(
+        Directory.projectRoot,
+        'chains',
+        this.originChain,
+        this.chain,
+        'bootnodes',
+      )
+      : path.join(
+        Directory.projectRoot,
+        'chains',
+        this.chain,
+        'bootnodes',
+      );
+
+    if (this.bootNodesFile) {
+      bootNodePath = this.bootNodesFile;
+      this.logInfo(`Reading bootnodes from file ${bootNodePath}`);
+    }
+
     try {
       this.logInfo('reading bootnodes from disk');
-      this.bootnodes = fs.readFileSync(
-        path.join(
-          Directory.projectRoot,
-          'chains',
-          this.originChain,
-          this.chain,
-          'bootnodes',
-        ),
+      this.bootnodes = `${fs.readFileSync(
+        bootNodePath,
```

Comment:
for dev chains `bootNodePath` wouldn't exist. and no need to publish a fresh image on dockerhub also ?

Reply:
if yes, small check on existence of `bootNodePath` could be added